backend/src/src-licenseinfo/src/main/java/com/siemens/sw360/licenseinfo/LicenseInfoHandler.java

```diff
@@ -35,30 +53,192 @@
  */
 public class LicenseInfoHandler implements LicenseInfoService.Iface {
 
-    private final AttachmentDatabaseHandler attachmentDatabaseHandler;
+    public static final String LICENSE_INFOS_CONTEXT_PROPERTY = "licenseInfos";
+    public static final String LICENSES_CONTEXT_PROPERTY = "licenses";
+    public static final String LICENSE_INFO_TEMPLATE_FILE = "src/main/resources/licenseInfoFile.vm";
```

Comment:
I am confused, i think the IDE has made this change while refactoring the location of `licenseInfoFile.vm`